client/src/store/modules/eventType/actions.js

```diff
@@ -0,0 +1,72 @@
+import * as actions from './types/actions';
+import * as mutations from './types/mutations';
+import eventTypeService from '@/services/eventType/eventTypeService';
+import { SET_ERROR_NOTIFICATION } from '@/store/modules/notification/types/actions';
+import { eventTypeMapper } from '@/services/Normalizer';
+
+export default {
+    [actions.GET_EVENT_TYPE_BY_ID]: async (context, id) => {
+        try {
+            const eventType = await eventTypeService.getEventTypeById(id);
+            context.commit(mutations.SET_EVENT_TYPE, eventType);
+
+            return Promise.resolve(eventTypeMapper(eventType));
```

Comment:
actually, you do not need to return data, because according to flux pattern, after you update state, your component will be re-rendered, returning data from action breaks the pattern